lib/susedistribution.pm

```diff
@@ -96,14 +96,17 @@ sub x11_start_program($$$) {
         send_key('alt-t');
         sleep 3;
     }
-    send_key('ret', 1);
+    send_key "ret";
+    wait_still_screen;
     # make sure desktop runner executed and closed when have had valid value
```

Comment:
you can ask the ones that have been working on this in before.
@dzedro, @nilxam can you help @kravciak with his questions?